backend/oasst_backend/config.py

```diff
@@ -33,6 +33,7 @@ def assemble_db_connection(cls, v: Optional[str], values: Dict[str, Any]) -> Any
 
     BACKEND_CORS_ORIGINS: List[AnyHttpUrl] = []
     UPDATE_ALEMBIC: bool = True
+    USE_SEED_DATA: bool = True
```

Comment:
could we set this to false, but set to true in the docker-compose?

Reply:
good idea, will do.

Reply:
yes, is done.